src/SFA.DAS.EmployerApprenticeshipsService.Application/SFA.DAS.EAS.Application.csproj

```diff
@@ -383,8 +383,6 @@
     <Compile Include="Mappings\LegalEntityMappings.cs" />
     <Compile Include="Queries\Extensions\QueryExtensions.cs" />
     <Compile Include="Queries\GetLegalEntity\GetLegalEntityQuery.cs" />

```

Comment:
Can the query be removed from this project?

Reply:
Yes it could; I left it as I was just doing C&P, but could turn it into primitive parameters is we want to do that as well.

Reply:
Updated to primitive parameters.